rest-api/dao/participant_summary_dao.py

```diff
@@ -514,6 +514,43 @@ def calculate_max_core_sample_time(self, participant_summary, field_name_prefix=
     else:
       return None
 
+  def calculate_distinct_visits(self, session, pid):
+    """ Participants may get PM or biobank samples on same day. This should be considered as
+    a single visit in terms of program payment to participant.
+    return Boolean: true if there has not been an order on same date (site timezone)."""
+    sql = text("""select CURDATE() as realdate , pm.created, bo.created,
```

Comment:
We should revisit this soon to take in a date as a parameter so that we can extend the tests.

Reply:
agreed, or mock the query ? I'll make a ticket.